[examples]/test_vehicles/s_test_cmd.lua

```diff
@@ -19,3 +19,24 @@ addCommandHandler("myvehicle", function(player)
         outputChatBox("This vehicle has the custom model ID " .. customModel .. ", which is based on the default model ID " .. baseModel .. " ("..(tostring(getVehicleNameFromModel(baseModel)) or "")..")", player, 0, 255, 0)
     end
 end, false, false)
+
+addEvent("spawnVehicleServer", true)
```

Comment:
Good job, thank you.
Suggestion: please rename all custom events you added to include a custom prefix like
`newmodels-test_vehicles:EVENT_NAME`
I like doing this, it's a good practice to distinguish where the custom events come from